client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -203,18 +241,15 @@ const tryGetLabelRates = ( orderId, siteId, dispatch, getState ) => {
 	}
 
 	const formState = getShippingLabel( state, orderId, siteId ).form;
```

Comment:
`getShippingLabel` can return null ([ref](https://github.com/Automattic/wp-calypso/blob/9d78dbb5d3015931691c4d28423b3ce9c10e16c0/client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js#L23)). We should handle this error - perhaps by aborting & showing a notice to the user?

Reply:
Same answer as [here](https://github.com/Automattic/wp-calypso/pull/24974#discussion_r205111482). If `getShippingLabel()` returned `null`, the modal wouldn't render and this code wouldn't run.